edm4eic.yaml

```
@@ -140,6 +140,13 @@ components:
       - edm4eic::Cov2f    directionError  // Error on the polar and azimuthal angles
       - float             pathlength      // Pathlength from the origin to this point
       - float             pathlengthError // Error on the pathlenght
+
+  ## PID hypothesis from Cherenkov detectors
+  edm4eic::CherenkovPdgHypothesis:
```

Not too keen on the `Pdg` which doesn't seem like consistent capitalization for an acronym. It's also not really only PDG anymore, it's any PID code.

How about
```suggestion
  edm4eic::CherenkovPIDHypothesis:
```

Or
```suggestion
  edm4eic::CherenkovParticleIDHypothesis:
```

---

Personally prefer `PID`, but going with `ParticleID` for consistency.